src/tooltip.tsx

```diff
@@ -0,0 +1,151 @@
+import isString from 'lodash/isString';
+import merge from 'lodash/merge';
+import * as React from 'react';
+import styled from 'styled-components';
+import { TooltipPlacement } from './common-types';
+import RCTooltip from 'rc-tooltip';
+
+// The time in ms to show a tooltip after being triggered by a click event
+const CLICK_TIMEOUT = 1000;
+
+const Arrow = styled.div`
```

Comment:
Are these from the arrow from the previous tooltip? Let's just use some library or whatever is default from rc-tooltip, and improve it later. I would really prefer to keep the codebase size to a minimum, especially for some trivial elements such as an arrow.

Reply:
@sradevski, sure inwhich case we will have to import rc-tooltips css too. I didn't do it, since we wanted to have our own design.

Reply:
@iamsolankiamit we might want to check with Alexandra on an updated design for the tooltips, but let's just start with the defaults first. Let's make this functional first, and afterwards please chat with Alexandra on providing you with a design that would match the rest of the designs better.